src/schema/tags.ts

```diff
@@ -106,11 +121,25 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
 
         return builder;
       }),
+    trendingTags: async (source, args, ctx): Promise<GQLTag[]> => {
+      // TODO: Implement query based on new MV
+      const { limit = 10 } = args;
+      const hits = await ctx.getRepository(Keyword).find({
+        select: ['value'],
+        order: { value: 'ASC' },
+        where: { status: 'allow' },
+        take: limit,
+      });
+      return hits.map((x) => ({ name: x.value }));
```

Comment:
```suggestion
      const keywords = await ctx.getRepository(Keyword).find({
        select: ['value'],
        order: { value: 'ASC' },
        where: { status: 'allow' },
        take: limit,
      });
      return keywords.map((keyword) => ({ name: keyword.value }));
```

Just a personal preference, but I prefer naming the variables to be similar to what we're fetching/looping over

Reply:
Ah this will be completely removed anyway :) 
It's just the old placeholder for local dev until we have the MV